### PR TITLE
Enable Elasticsearch client on mon01

### PR DESCRIPTION
**Pull request by thouveng**

### CODE REVIEW

**Comment by thouveng** [comment]
This change requires https://github.com/tcpcloud/salt-formula-elasticsearch/pull/9

**Comment by elemoine** [comment]
Is a similar change required for stacklight-basic as well?

**Comment by thouveng** [comment]
Yes probably because we will use the client to configure index templates. So it will be needed by basic too.

File: classes/cluster/mk20_stacklight_advanced/stacklight/client.yml [hunk 1]
```diff
@@ -4,3 +4,4 @@ classes:
 - system.heka.metric_collector.single
 - cluster.mk20_stacklight_advanced.stacklight
 - service.grafana.collector
+- system.elasticsearch.client.single
```
Comment (simonpasquier) [inline classes/cluster/mk20_stacklight_advanced/stacklight/client.yml]: should be removed?

**Comment by thouveng** [comment]
Yes I removed it

File: classes/cluster/mk20_stacklight_basic/stacklight/client.yml [hunk 2]
```diff
@@ -4,3 +4,4 @@ classes:
 - system.heka.metric_collector.single
 - cluster.mk20_stacklight_basic.stacklight
 - service.grafana.collector
+- system.elasticsearch.client.single
```
Comment (elemoine) [inline classes/cluster/mk20_stacklight_basic/stacklight/client.yml]: Should be in `server.yml`.

Comment (thouveng) [inline classes/cluster/mk20_stacklight_basic/stacklight/client.yml]: Yes. I will do for basic what I did for advanced.

File: classes/cluster/mk20_stacklight_advanced/stacklight/client.yml (hunk 1 above)
Comment (thouveng) [inline classes/cluster/mk20_stacklight_advanced/stacklight/client.yml]: done

File: classes/cluster/mk20_stacklight_advanced/fuel/config.yml [hunk 3]
```diff
@@ -52,6 +52,9 @@ parameters:
           - service.galera.slave.cluster
           params:
             mysql_cluster_role: slave
+        stacklight_server_node01:
+          classes:
+            - system.elasticsearch.client.single
```
Comment (elemoine) [inline classes/cluster/mk20_stacklight_advanced/fuel/config.yml]: You need to do the same in mk22_lab_advanced.

Comment (elemoine) [inline classes/cluster/mk20_stacklight_advanced/fuel/config.yml]: You need to do the same in mk22_lab_advanced.

Comment (thouveng) [inline classes/cluster/mk20_stacklight_advanced/fuel/config.yml]: done

**Pull request merged**